bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -410,6 +410,10 @@
 		<label>Activate Binding Control</label>
 		<description>Activate (control through openHAB) or deactivate (return control to CCEase) binding control</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Opening</tag>
```

Comment:
As this just enables the control via binding, I would rather use `Enabled` here. `Opening` is definitely not suitable.

Reply:
`Enabled` doesn't exist so I will change to `Mode` which is probably the closest.

Reply:
Huh, but it's in the .csv...
https://github.com/jimtng/openhab-core/blob/685264295c62f5e8179d36ba06b60aa55cc9115f/bundles/org.openhab.core.semantics/model/SemanticTags.csv#L63
Or is it that it just can't be used together with `Switch` (but I can't figure how this is indicated)?

BTW, I think `Control` would fit better than `Switch` here. And I don't see a necessity to add a tag here, if there is no perfect match (`Control` or `Switch` + `Enabled` I would consider as perfect match).

Reply:
Oops. My mistake. Changed to `Enabled`